spring-cloud-alibaba-examples/pom.xml

```diff
@@ -56,6 +56,27 @@
 		<module>integrated-example/integrated-frontend</module>
 	</modules>
 
+    <properties>
+        <seata.version>1.6.1-SNAPSHOT</seata.version>
+        <nacos-client.version>1.4.2</nacos-client.version>
+    </properties>
+
+    <dependencyManagement>
```

Comment:
It just temporary dependencies, we should don't add them in the PR. Maybe when we need to test and we add them manually.